internal/kubeconfig/provider.go

```diff
@@ -106,9 +114,42 @@ func (p *FakeProvider) K8sClientForRuntimeID(_ string) (client.Client, error) {
 }
 
 func (p *FakeProvider) KubeconfigForRuntimeID(runtimeID string) ([]byte, error) {
-	return []byte("fake kubeconfig"), nil
+	return []byte(`
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: ca
+    server: https://my.cluster
+  name: <cluster-name>
+contexts:
+- context:
+    cluster:  cname
+    user:  cuser
+  name:  cname
+current-context:  cname
+kind: Config
+preferences: {}
+users:
+- name:  cuser
+  user:
+    token: some-token
+`), nil
+}
+
+func (p *FakeProvider) K8sClientSetForRuntimeID(runtimeID string) (v12.CoreV1Interface, rbac.RbacV1Interface, error) {
+	return p.coreClient, p.rbacClient, nil
 }
 
-func (p *FakeProvider) K8sClientSetForRuntimeID(runtimeID string) (*kubernetes.Clientset, error) {
-	return nil, fmt.Errorf("not implemented")
+func createFakeClients() (v12.CoreV1Interface, rbac.RbacV1Interface) {
```

Comment:
What do you think about using fake clientset from client-go?

Reply:
changed to kubernetes.Interface